app/src/main/java/org/simple/clinic/editpatient/EditPatientInit.kt

```diff
@@ -21,7 +24,7 @@ class EditPatientInit(
         LoadInputFields,
     )
 
-    if (!model.hasColonyOrVillagesList) {
+    if (!model.hasColonyOrVillagesList && features.isEnabled(Feature.VillageTypeAhead)) {
```

Comment:
Can we verify this behaviour in tests?

Reply:
We test feature flags in our tests as well? 🤔

Reply:
Depends on the feature flag itself, I guess.

Reply:
I pushed other changes, but if you can confirm if this is something we need for this feature flag I am gonna add this case and push again or else the rest of the changes are done.

Reply:
Let's skip this for now and add the test in another PR.